### PR TITLE
sync: update web/index

### PR DESCRIPTION
**Pull request by jasonren0403**
Sync with the newest English document of web/index

### CODE REVIEW

File: files/zh-cn/web/index.md [hunk 1]
```diff
@@ -10,22 +10,53 @@ slug: Web
...
 - [面向 Web 开发者的教程](/zh-CN/docs/Web/Tutorials)
   - : 这些教程帮助你一步一步地学习 API、技术或更广泛的主题。
-- [渐进式 Web 应用（PWAs）](/zh-CN/docs/Web/Progressive_web_apps)
-  - : 渐进式 Web 应用是使用新兴 Web 浏览器 API 和功能以及运用传统渐进增强策略的 Web 应用，为跨平台 Web 应用带来类似原生应用的用户体验。
+- [无障碍](/zh-CN/docs/Web/Accessibility)
+  - : 让更多人能够使用你的网站，即使他们的能力在某种程度上受限。
+- [性能](/zh-CN/docs/Web/Performance)
+  - : 让网站上的内容尽可能快地发挥可用性和交互性。
+- [安全](/zh-CN/docs/Web/Security)
+  - : 保护用户免受数据泄露和偷窃、侧信道攻击和跨站脚本、内容注入和点击劫持之害。
```
Comment (jasonren0403) [inline files/zh-cn/web/index.md:18]:
```suggestion
  - : 保护用户免受数据泄露和偷窃、侧信道攻击、跨站脚本、内容注入和点击劫持之害。
```

File: files/zh-cn/web/index.md [hunk 2]
```diff
@@ -10,22 +10,53 @@ slug: Web
   - : Web 开发者指导提供手把手教学，帮助你实际使用 Web 技术做你想做或需要做的。
 - [面向 Web 开发者的教程](/zh-CN/docs/Web/Tutorials)
   - : 这些教程帮助你一步一步地学习 API、技术或更广泛的主题。
-- [渐进式 Web 应用（PWAs）](/zh-CN/docs/Web/Progressive_web_apps)
-  - : 渐进式 Web 应用是使用新兴 Web 浏览器 API 和功能以及运用传统渐进增强策略的 Web 应用，为跨平台 Web 应用带来类似原生应用的用户体验。
+- [无障碍](/zh-CN/docs/Web/Accessibility)
+  - : 让更多人能够使用你的网站，即使他们的能力在某种程度上受限。
```
Comment (yin1999) [inline files/zh-cn/web/index.md:14]:
```suggestion
  - : 让更多人能够使用你的网站，即使他们的某些能力受限。
```

File: files/zh-cn/web/index.md [hunk 3]
```diff
@@ -10,22 +10,53 @@ slug: Web
...
 - [SVG](/zh-CN/docs/Web/SVG)
   - : 可缩放矢量图形（Scalable Vector Graphics）使你能够使用一组矢量与形状来描述图片，在显示时可提供无关大小的零失真的平滑缩放。
 - [MathML](/zh-CN/docs/Web/MathML)
   - : Mathematical 标记语言让显示复杂的数学公式或语法成为可能。
 - [Web Components](/zh-CN/docs/Web/Web_Components)
   - : Web Components 允许你在自己的应用中创建和复用自定义元素。
+- [WebDriver](/zh-CN/docs/Web/WebDriver)
+  - : WebDriver 是一种浏览器自动化机制，通过模拟真实的人使用浏览器的动作来远程控制浏览器。它被广泛用于网络应用的跨浏览器测试。
+- [Web 扩展](/zh-CN/docs/Mozilla/Add-ons/WebExtensions)
+  - : 网络扩展是一种在浏览器中为用户提供增强功能的方式——用于阻止广告和其他内容、定制页面的外观以及更多行为。
```
Comment (yin1999) [inline files/zh-cn/web/index.md:47]:
```suggestion
  - : Web 扩展是一种在浏览器中为用户提供增强功能的方式——用于阻止广告和其他内容、定制页面的外观以及更多行为。
```

File: files/zh-cn/web/index.md [hunk 4]
```diff
@@ -10,22 +10,53 @@ slug: Web
...
+## 开发者工具文档
+
+- [Firefox 开发者工具](https://firefox-source-docs.mozilla.org/devtools-user/index.html)
+  - : Firefox 自带的网络开发者工具的文档。
+- [Chrome 开发者工具](https://developer.chrome.com/docs/devtools/)
+  - : Chrome 自带的网络开发者工具的文档。
+- [Safari Web 监视器](https://webkit.org/web-inspector/)
+  - : Safari 自带的网络开发者工具的文档。
+- [Edge 开发者工具](https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/landing/)
+  - : Edge 自带的网络开发者工具的文档。
```
Comment (yin1999) [inline files/zh-cn/web/index.md:62]: better?

```suggestion
  - : Firefox 内置的网络开发者工具的文档。
- [Chrome 开发者工具](https://developer.chrome.com/docs/devtools/)
  - : Chrome 内置的网络开发者工具的文档。
- [Safari Web 监视器](https://webkit.org/web-inspector/)
  - : Safari 内置的网络开发者工具的文档。
- [Edge 开发者工具](https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/landing/)
  - : Edge 内置的网络开发者工具的文档。
```

**Review by yin1999 — APPROVED** [review]
LGTM, some suggestions

File: files/zh-cn/web/index.md [hunk 5]
```diff
@@ -10,22 +10,53 @@ slug: Web
...
   - : 层叠样式表用来描述 Web 内容的外观。
 - [JavaScript](/zh-CN/docs/Web/JavaScript)
   - : JavaScript 是一种用于为网站增加交互特性的编程语言。
+- [WebAssembly](/zh-CN/docs/WebAssembly)
+  - : WebAssembly 技术允许使用 C、C++、Rust、Swift、C#、Go 等语言编写的程序在 Web 上运行。
+- [事件](/zh-CN/docs/Web/Events)
+  - : 事件是 Web 应用的反馈；例如，当一个网页完成加载，或者用户选择什么，按下一个键，调整一个窗口的大小，提交一个表单，或者暂停一个视频。
+- [HTTP](/zh-CN/docs/Web/HTTP)
+  - : HTTP是一种基本的互联网协议，用于在网络上获取文档、样式表、脚本、图像、视频、字体和其他资源，并将数据送回到网络服务器。
+- [Media](/zh-CN/docs/Web/Media)
```
Comment (jasonren0403) [inline files/zh-cn/web/index.md:36]:
```suggestion
- [媒体](/zh-CN/docs/Web/Media)
```

Comment (jasonren0403) [inline files/zh-cn/web/index.md:36]:
```suggestion
- [媒体技术](/zh-CN/docs/Web/Media)
```

**Pull request merged**